### PR TITLE
feature(form): add formulas on table

### PR DESCRIPTION
**Pull request by peppedeka**

### CODE REVIEW

File: src/core/forms/form-renderer.ts [hunk 1]
```diff
@@ -84,6 +84,7 @@ import {orderedNodes} from './utils/nodes/ordered-nodes';
 import {updateRepsNum} from './utils/slides-instances/update-reps-num';
 import {validSlide} from './utils/slides-instances/valid-slide';
 import {AjfValidationService} from './validation-service';
+import {AjfTableCell} from '.';
```
Comment (trik) [inline src/core/forms/form-renderer.ts]: please don't import from local barrel, specify import file (and sort inputs)

File: src/core/forms/utils/fields-instances/create-table-field-instance.ts [hunk 2]
```diff
@@ -24,17 +24,37 @@ import {AjfContext} from '@ajf/core/models';
 
 import {AjfTableFieldInstance} from '../../interface/fields-instances/table-field-instance';
 import {AjfFieldInstanceCreate, createFieldInstance} from './create-field-instance';
+import {AjfTableField, AjfTableCell} from '../..';
```
Comment (trik) [inline src/core/forms/utils/fields-instances/create-table-field-instance.ts]: please don't import from package barrel, use full relative path (and sort inputs)

File: src/core/forms/form-renderer.ts (hunk 1 above)
Comment (peppedeka) [reply src/core/forms/form-renderer.ts]: done

File: src/core/forms/utils/fields-instances/create-table-field-instance.ts (hunk 2 above)
Comment (peppedeka) [reply src/core/forms/utils/fields-instances/create-table-field-instance.ts]: done

**Pull request merged**